force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -44,9 +58,21 @@ public with sharing class VariablesDataTypesOperators {
      * @return The difference between the two numbers.
      */
     public static Integer subtraction(Integer a, Integer b) {
-        return null; // Replace null with the variable you used to store the result
-    }
-
+        Integer difference = 0;
+        
+         if (a != null && b != null) {
+          difference = a - b;
+       //  System.debug(difference);
+         return difference; // Replace null with the variable you used to store the result
+     }
+     else if 
+          (a == null || B == null) {
+        // System.debug('Values are null');
+          Return null;
+     } 
+     return difference;
+       }
+     // subtraction(3,5);
```

Comment:
in order to line up your brackets, I suggest you download the vs code extension "indent-rainbow"